test/index.js

```diff
@@ -105,6 +105,13 @@ describe('APIs', function () {
       expect(apis.Session.description).to.equal('Get and set properties of a session.')
     })
 
+    they('include code in description', function () {
+      var properties = apis.BrowserWindow.constructorMethod.parameters[0].properties
+      for (var property of properties) {
+        expect(property.description).to.not.include('Default is .')
```

Comment:
The quote `Default is .` appears a lot in our API json at the moment because typically in the docs we have

```markdown
Default is `true`.
```

I guess this is just asserting that we aren't stripping that default thing in backticks